config/config.exs

```diff
@@ -16,7 +16,8 @@ config :castle, CastleWeb.Endpoint,
   secret_key_base: "+DVZXtoG6yRaQrhCNCPNjdyhioRgRlrKMUDDlZkPLXCghP4NCJ+JafxydZD/QnOq",
   render_errors: [view: CastleWeb.ErrorView, accepts: ~w(html json hal)],
   pubsub: [name: Castle.PubSub,
-           adapter: Phoenix.PubSub.PG2]
+           adapter: Phoenix.PubSub.PG2],
+  http: [compress: true]
```

Comment:
Does this gzip everything?  Or only if the client indicates they accept gzipped responses?

Reply:
Seems like the latter, via `curl -H "Accept-Encoding: gzip"`.  Thanks for doing the right thing, phoenix!